client/app/index.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable */
```

Comment:
Could we disable only affected rules in that case?

Reply:
done